src/utils/url.ts

```diff
@@ -0,0 +1,3 @@
+export function wrapUtm(url: string) {
+  return `${url}?utm_source=email`;
```

Comment:
please add `utm_medium=email` too
actually `email` should be medium, and source should be the actual email sent e.g. `notification_mail_a`, but we can stick with `source=email` if edmond likes it

Reply:
Updated